app/services/unpublish_service.rb

```diff
@@ -88,6 +86,21 @@ def remove_and_redirect(edition, redirect_path, explanatory_note: nil)
 
 private
 
+  def create_withdrawal(edition, public_explanation)
```

Comment:
I thought we already allowed people to edit the withdrawal explanation? I can't see a commit for this, so just wondering how this came about...

Reply:
We do, well we allow them to edit a withdrawal on the live site and the banner but it creates a new withdrawal and status in the app, here i'm only changing the process of submitting / editing a withdrawal to include saving the `published_status` of the document

Reply:
Oh yeah, I see :-).